recipes/self-managed-otlp-ingest/README.md

```diff
@@ -0,0 +1,150 @@
+```
+
+Connect `kubectl` to the created cluster:
+```shell
+gcloud container clusters get-credentials ${CLUSTER_NAME} --region ${CLUSTER_REGION} --project ${PROJECT_ID}
+```
+
+### Deploy sample application (Java) on your cluster
+
+Deploy an un-instrumented application on your GKE cluster. The application used here is the un-instrumented version of the [Java Instrumentation Quickstart](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/tree/main/examples/instrumentation-quickstart).
+
+To build the sample app, from the root of this recipe:
+```shell
+# We pull the application from GitHub and build it 
+git clone https://github.com/GoogleCloudPlatform/opentelemetry-operations-java.git
+pushd opentelemetry-operations-java/examples/instrumentation-quickstart && \
+DOCKER_BUILDKIT=1  docker build -f uninstrumented.Dockerfile -t java-quickstart . && \
+popd && \
+rm -rf opentelemetry-operations-java
+```
+
+Next, push the locally built image to Google Artifact Registry:
+```shell
+# Export environment variables for configuring Artifact Registry
+export CONTAINER_REGISTRY=<your-registry-name>
+export REGISTRY_LOCATION=us-central1
+
+# If you do not have an Artifact repository, make one:
+gcloud artifacts repositories create ${CONTAINER_REGISTRY} --repository-format=docker --location=${REGISTRY_LOCATION} --description="Sample applications to auto-instrument using OTel operator"
+# Alternatively, you can use any existing artifact registry
+
+docker tag java-quickstart:latest ${REGISTRY_LOCATION}-docker.pkg.dev/${PROJECT_ID}/${CONTAINER_REGISTRY}/java-quickstart:latest
+docker push ${REGISTRY_LOCATION}-docker.pkg.dev/${PROJECT_ID}/${CONTAINER_REGISTRY}/java-quickstart:latest
+```
+
+Finally, deploy the pushed application in your cluster:
+```shell
+# This relies on the environment variables exported in previous steps
+envsubst < k8s/quickstart-app.yaml | kubectl apply -f -
+```
+
+Once your application is deployed, you can interact with it by hitting either `/single` or `/multi` endpoints on port `8080`. For more information about the application, view the [application readme](uninstrumented-app/examples/instrumentation-quickstart/README.md).\
+*Note: You may need to configure port forwarding to interact with this application over localhost.*
+
+## Instrumenting deployed applications
+
+Now that we have an application deployed on a GKE cluster, we can instrument the application using the OpenTelemetry operator.
+
+### Installing OpenTelemetry Operator
+
+We use the OpenTelemetry Operator to inject auto-instrumentation into the apps running in our cluster.
+Follow [these steps](../../README.md#prerequisites) to ensure that `cert-manager` is installed on your cluster.
+
+```shell
+# Install OpenTelemetry Operator
+kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/download/v0.116.0/opentelemetry-operator.yaml
+```
+
+### Deploying a self-managed OpenTelemetry Collector on the cluster
```

Comment:
Deploying the collector should be separate from instrumenting applications.  I.e.

Step 1: deploy sample app
Step 2: Deploy opentelemetry collector
step 3: Instrument application.

Reply:
I will move the collector deployment to a separate heading instead of sub-heading to indicate it is a distinct, unrelated step.